libdnf/base/goal.cpp

```diff
@@ -307,17 +307,17 @@ GoalProblem Goal::Impl::add_specs_to_goal() {
             case Action::UPGRADE_ALL: {
                 rpm::SolvQuery query(&sack);
                 rpm::solv::IdQueue upgrade_ids;
-                for (auto package_id : *query.p_impl) {
-                    upgrade_ids.push_back(package_id.id);
+                for (auto id : *query.p_impl) {
+                    upgrade_ids.push_back(id);
                 }
                 bool clean_requirements_on_remove = settings.clean_requirements_on_remove == GoalSetting::SET_TRUE;
                 rpm_goal.add_upgrade(upgrade_ids, settings.get_best(cfg_main), clean_requirements_on_remove);
             } break;
             case Action::DISTRO_SYNC_ALL: {
                 rpm::SolvQuery query(&sack);
                 rpm::solv::IdQueue upgrade_ids;
-                for (auto package_id : *query.p_impl) {
-                    upgrade_ids.push_back(package_id.id);
+                for (auto id : *query.p_impl) {
```

Comment:
[Not important] The name of variable can remain unchanged. It contains package id.